### ISSUE TITLE
discuss: apisix supports obtaining data plane information

### ISSUE THREAD

**Issue by Firstsawyou** [apache/apisix#2453]
### Issue description

When we obtain the data plane information, we need to obtain it through the `curl` method, and it is more difficult to obtain multiple pieces of data information (e.g: obtain 1000 `route` data). Can we achieve such a function through plugin。

**Comment by membphis** [comment]
what means the `data plane information`? monitor data?

**Comment by Firstsawyou** [comment]
No, it is the data stored by etcd. For example, `/apisix/routes/1` means `route` data whose route id is `1`.

**Comment by membphis** [comment]
I got your mind. thx for your explain
